versions.props

```diff
@@ -9,7 +9,7 @@ com.google.dagger:dagger = 2.0.2
 com.google.dagger:dagger-compiler = 2.0.2
 com.google.guava:* = 18.0
 com.googlecode.json-simple:json-simple = 1.1.1
-com.jayway.awaitility:awaitility = 1.6.5
```

Comment:
Looked through the release notes, https://github.com/awaitility/awaitility/wiki/ReleaseNotes30. The changes make sense. Thanks for the contribution!